### ISSUE TITLE
Missing information about the frame_delay in movie/config

### ISSUE THREAD

**Issue by magicus** [scrool/xled-docs#10]
I'm trying to figure out the meaning of the `frame_delay` parameter for movie/config.

I've tried sending various values, but can't really understand what this means. Obviously it affects the frame rate, but how?

It seems non-integer values are rejected, and so are negative numbers as well as zero.

I think the highest FPS (frames per second) was achieved on my device with a value of about 150, which just caused a flickering too quick for me to be able to count. The lowest value was 1, which was about 1-2 FPS.

However, it's not a simple linear conversion. Setting the frame_delay to 2 caused an insane flickering, like above 150. 

I created a movie with two frames, one entirely blue and one green, and tried running it with different frame_delay. I counted 30 "blinks" (that is, when two frames has passed and the movie loops) and tried clocking it. (I could hardly clock with second precision, and might have missed frames when counting, so this is approximate.)

| frame_delay | time for 30 frames | 
|-------------|--------------------|
| 1     | 32 s 
| 2    | uncountable,  just flickering fast
| 3   | 90 s
| 4   | 13 s
| 10  | 25 s
| 50 | 12 s 
| 100 | flickering
| 150 | flickering even faster

To me, this makes no sense at all. I think it would be valuable to document this value properly, but the first step is to figure out how this actually works. I hope this could be the start of discussion.

**Comment by magicus** [comment]
I approached this the wrong way. The timing I saw was probably just artifacts, like some kind of wagon-wheel effect, since the numbers I tested was far too low.

The actual fact is this:

The `frame_delay` is measured in milliseconds, and is the time the device will wait after updating a frame until showing the next. So to get a slideshow with a new frame every fifth second, a value of `5000` should be sent, and to get 25 frames per second (FPS), a value of `40` should be sent. In general, to get `x` FPS, a value of `1000 / x` should be sent.
